server.go

```diff
@@ -1038,11 +1059,6 @@ func (s *server) Start() error {
 		srvrLog.Infof("Auto peer bootstrapping is disabled")
 	}
 
-	// Start a goroutine that will periodically send out ChannelUpdates
-	// based on a channel's status.
-	s.wg.Add(1)
-	go s.watchChannelStatus()
```

Comment:
Just remove the method as well?

Reply:
squashed the removal into the commit where everything is hooked up